webapp/src/main/resources/public/js/components/workbench/GitBlameInfoModal.js

```diff
@@ -271,7 +293,7 @@ let GitBlameInfoModal = React.createClass({
      */
     getLocationExtractorPrefix() {
         try {
-            return this.props.appConfig.link.link[this.props.textUnit.getRepositoryName()].location.extractorPrefix;
```

Comment:
does this work without the second `link`? 😮

Reply:
see comment on `@JsonUnwrapped`